backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java

```diff
@@ -25,4 +27,12 @@ public BucketResponse getBucket(UUID uuid) {
     BucketResponse bucketResponse = modelMapper.map(bucketEntity, BucketResponse.class);
     return bucketResponse;
   }
+
+  @Override
+  public List<BucketResponse> listAllBuckets() {
+    List<BucketEntity> entityList = (List<BucketEntity>) bucketsRepository.findAll();
```

Comment:
findAll() returns an Iterable? I can't believe this works

nice

Reply:
Yes, it works like this :)